### DISCUSSION TITLE
我一不小心把recaptcha的服务端和用户端密钥配反了。。。

### DISCUSSION THREAD

**Discussion by ning0818** [Qexo/Qexo#401] [Q&A] [answered]
我一不小心把recaptcha的服务端和用户端密钥配反了。。。

现在一登录就一直这样
![image](https://github.com/Qexo/Qexo/assets/117955401/c792cc6f-53c1-4351-b464-0a958db90402)

请问在哪里能改回来？
![image](https://github.com/Qexo/Qexo/assets/117955401/2e64efcd-2c49-41f6-97a2-8e4b1bddc45e)

**Comment by am-abudu** [comment, marked as answer]
在数据库删除包含RECAPTCHA的字段即可

**Reply by ning0818** [reply]
秒回啊！！！我试试

**Reply by ning0818** [reply]
咋删除？
![image](https://github.com/Qexo/Qexo/assets/117955401/9f539b0f-9969-457e-8d59-e5f29a27c16f)

**Reply by am-abudu** [reply]
数据库 找到包含settingmodel的表 然后找到字段里面包含RECAPTCHA的项 删除即可

**Reply by ning0818** [reply]
谢谢
![image](https://github.com/Qexo/Qexo/assets/117955401/2bcc73cc-6f7a-43d2-aff1-c35a2ef46f1c)
